custom_plugins/stream_overlays/static/css/apex/trackdraw_map.css

```
@@ -0,0 +1,35 @@
+@import url("https://fonts.googleapis.com/css2?family=Rajdhani:wght@500;600;700&display=swap");
```

Stylelint (17.9.0)

[error] Expected "url("https://fonts.googleapis.com/css2?family=Rajdhani:wght@500;600;700&display=swap")" to be ""https://fonts.googleapis.com/css2?family=Rajdhani:wght@500;600;700&display=swap"" (import-notation)

(import-notation)